packages/sdk/tests/test-utils.ts

```diff
@@ -2,10 +2,10 @@ import { jest } from '@jest/globals';
 import { Wallet, JsonRpcProvider, ethers } from 'ethers';
 import { IExecAppModule, IExecConfig, TeeFramework, utils } from 'iexec';
 import {
-  DataProtectorConfigOptions,
-  Web3SignerProvider,
-  getWeb3Provider,
+  type DataProtectorConfigOptions,
+  type Web3SignerProvider,
 } from '../src/index.js';
+import { getWeb3Provider } from '../src/utils/getWeb3Provider.js';
```

Comment:
Very important to **not** import this function from root `index.ts`.
Otherwise importing something from this `test-utils.ts` file would mean to import the whole SDK, which would break unit test mocks.